### PR TITLE
Added mutation example with arguments #289 #65

### PR DESCRIPTION
**Pull request by Anton-4**

### CODE REVIEW

File: examples/src/Example09BasicMutation.elm [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+chatSelection : SelectionSet ChatMessage Object.ChatMessage
+chatSelection =
+    SelectionSet.map ChatMessage
+        ObjChat.phrase
+
+
+sendChatMutation : SelectionSet Response RootMutation
+sendChatMutation =
+    Mutation.sendMessage
+        (SendMessageRequiredArguments (CustomScalarCodecs.Id 1001) Droids)
```
Comment (dillonkearns) [inline examples/src/Example09BasicMutation.elm:34]: Using record literal syntax is easier to read here.

```suggestion
        { characterId = CustomScalarCodecs.Id 1001, phrase = Droids }
```

File: examples/src/Example09BasicMutation.elm [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+
+type alias Response =
+    Maybe ChatMessage
+
+
+chatSelection : SelectionSet ChatMessage Object.ChatMessage
+chatSelection =
+    SelectionSet.map ChatMessage
```
Comment (dillonkearns) [inline examples/src/Example09BasicMutation.elm:27]: The record seems unnecessary with just a single value. How about also adding the character, like in this example:

https://github.com/dillonkearns/elm-graphql/blob/master/examples/subscription/Main.elm#L34-L37

**Comment by Anton-4** [comment]
You're welcome, excellent suggestions :slightly_smiling_face:!

File: examples/src/Example09BasicMutation.elm [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    { phrase : Phrase
+    , character : Maybe Character
+    }
+
+
+type alias Response =
+    Maybe ChatMessage
+
+
+chatSelection : SelectionSet ChatMessage Object.ChatMessage
```
Comment (dillonkearns) [inline examples/src/Example09BasicMutation.elm:34]:
```suggestion
chatSelection : SelectionSet ChatMessage ChatMessage
```

File: examples/src/Example09BasicMutation.elm [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+import Graphql.Http
+import Graphql.Operation exposing (RootMutation)
+import Graphql.SelectionSet as SelectionSet exposing (SelectionSet)
+import Helpers.Main
+import RemoteData exposing (RemoteData)
+import Swapi.Enum.Phrase exposing (Phrase(..))
+import Swapi.Interface
+import Swapi.Interface.Character
+import Swapi.Mutation as Mutation exposing (SendMessageRequiredArguments)
+import Swapi.Object as Object exposing (ChatMessage)
```
Comment (dillonkearns) [inline examples/src/Example09BasicMutation.elm:14]:
```suggestion
import Swapi.Object exposing (ChatMessage)
```

**Pull request merged**